cmd/web-handlers.go

```diff
@@ -271,9 +271,10 @@ func (web *webAPIHandlers) ListBuckets(r *http.Request, args *WebGenericArgs, re
 		}
 		for _, dnsRecord := range dnsBuckets {
 			bucketName := strings.Trim(dnsRecord.Key, "/")
+
 			if globalIAMSys.IsAllowed(iampolicy.Args{
 				AccountName:     claims.Subject,
-				Action:          iampolicy.Action(policy.GetObjectAction),
+				Action:          iampolicy.ListBucketAction,
```

Comment:
Should this be `ListAllMyBucketsAction` ?

Reply:
no ListAllMyBucket means ListBuckets - ListBucket means ListObjects https://docs.aws.amazon.com/AmazonS3/latest/dev/using-with-s3-actions.html